eth-contracts/contracts/Governance.sol

```diff
@@ -201,10 +201,7 @@ contract Governance is InitializableV2 {
 
         // Require proposer is active Staker
         Staking stakingContract = Staking(stakingAddress);
-        require(
-            stakingContract.totalStakedFor(proposer) > 0,
-            "Proposer must be active staker with non-zero stake."
-        );
+        stakingContract.isCurrentStaker(proposer);
```

Comment:
shouldn't this still be a require statement?

Reply:
there's a require inside staking @hareeshnagaraj

Reply:
+1 to having helper return bool and calling `require(isCurrentStaker)`